ios/Podfile

```diff
@@ -64,10 +64,5 @@ target 'OneSignalNotificationServiceExtension' do
 end
 
 target 'HeliumWalletWidgetExtension' do
-  pod "Apollo", '0.53.0'
```

Comment:
Is there still code in the widget using apollo?

Reply:
Not anymore